src/callbacks_step/stepsize_dg1d.jl

```diff
@@ -3,87 +3,82 @@
 # we need to opt-in explicitly.
 # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
 @muladd begin
-
+#! format: noindent
 
 function max_dt(u, t, mesh::TreeMesh{1},
                 constant_speed::False, equations, dg::DG, cache)
-  # to avoid a division by zero if the speed vanishes everywhere,
-  # e.g. for steady-state linear advection
-  max_scaled_speed = nextfloat(zero(t))
-
-  for element in eachelement(dg, cache)
-    max_λ1 = zero(max_scaled_speed)
```

Comment:
Unrelated to the formatter, but I just saw this. Here, we use non-ASCII characters in executable code, which doesn't follow our style guide. This is done in all of the stepsize files.

Reply:
True, this is a remnant from the time before the style guide. If you want to change it in this PR, please go ahead 👍

Reply:
I created #1521.

Reply:
Thanks!